x-pack/plugins/ingest_manager/server/services/index.ts

```diff
@@ -24,12 +24,38 @@ export interface ESIndexPatternService {
  * A service that provides exported functions that return information about an Agent
  */
 export interface AgentService {
+  /**
+   * Get an Agent by id
+   */
+  getAgent(soClient: SavedObjectsClientContract, agentId: string): Promise<Agent>;
```

Comment:
When we use this method, we will have to look at the active property of the agent to know if it is unenrolled?

Reply:
Yes it's correct